src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -238,6 +239,33 @@ internal virtual PortViewModel CreateProxyPortViewModel(PortModel portModel)
             return new PortViewModel(node, portModel);
         }
 
+        private UIElement FindProxyPortUIElement(PortViewModel proxyPortViewModel)
+        {
+            var mainWindow = Application.Current.MainWindow;
+
+            return FindChild<UIElement>(mainWindow, e =>
+                e is FrameworkElement fe && fe.DataContext == proxyPortViewModel);
+        }
+
+        private T FindChild<T>(DependencyObject parent, Func<T, bool> predicate) where T : DependencyObject
```

Comment:
There should be a FindChild window method in WpfUtilities.cs. Can we reuse that.

Reply:
I don't think we can, the two are not equivalent. `PortViewModel.FindChild` returns specific `ContentPresenter` where the `DataContext` matches the `proxyPortViewModel` regardless of the `Name` property. Here `ContextPresenter.Name=""`. 
`GuideUtilities.FindChild` relies on the `Name` property and doesn't check for `DataContext`.
I prefer keeping the code as it is, unless you have any suggestions.

Reply:
Sounds good.